components/menu-items/item.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../button';
+import Shortcut from './shortcut';
+import IconButton from '../icon-button';
+import './style.scss';
+
+/**
+ * Renders a generic menu item for use inside the more menu.
+ *
+ * @return {WPElement} More menu item.
+ */
+function MenuItemsItem( { className, icon, label, onClick, shortcut, isSelected = false } ) {
+	className = classnames( 'components-menu-items__button', {
+		[ className ]: Boolean( className ),
```

Comment:
`classnames` does a truthiness check, so there's no need to coerce to a boolean value:

https://github.com/JedWatson/classnames/blob/34a05a53d31d35879ec7088949ae5a1b2224043a/index.js#L31

```js
className = classnames( 'components-menu-items__button', {
	[ className ]: className,
	'is-selected': isSelected,
} );
```

Further, this behavior is already accounted for by passing as an argument, since a class name is not included if it is falsey:

https://github.com/JedWatson/classnames/blob/34a05a53d31d35879ec7088949ae5a1b2224043a/index.js#L18

```js
className = classnames( 'components-menu-items__button', className, {
	'is-selected': isSelected,
} );
```

Reply:
Cool, I’ll update on Monday 👍

Reply:
Done in #5798.